spec/support/fetchable_behavior.rb

```diff
@@ -136,3 +173,45 @@
     end
   end
 end
+
+shared_examples_for 'a record with an indexable url' do
+  describe 'validations' do
+    it { is_expected.to allow_value("http://some.site.gov/url").for(:url) }
```

Comment:
We really only need one spec for a valid URL; the following two spec lines can be removed.